backend/app/api/routes/responses.py

```
@@ -21,8 +25,20 @@
+    if hasattr(e, "body") and isinstance(e.body, dict) and "message" in e.body:
         return e.body["message"]
+    elif hasattr(e, "message"):
+        return e.message
+    elif hasattr(e, "response") and hasattr(e.response, "json"):
+        try:
+            error_data = e.response.json()
+            if isinstance(error_data, dict) and "error" in error_data:
+                error_info = error_data["error"]
+                if isinstance(error_info, dict) and "message" in error_info:
+                    return error_info["message"]
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)